### PR TITLE
Fix: Some typos in streaming guides

### PR DESCRIPTION
**Pull request by freddyaboulton**
## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: demo/rt-detr-object-detection/run.py [hunk 1]
```diff
@@ -111,5 +111,10 @@ def stream_object_detection(video, conf_threshold):
         outputs=[output_video],
     )
 
+    gr.Examples(
+        examples=["3285790-hd_1920_1080_30fps.mp4"],
```
Comment (abidlabs) [inline demo/rt-detr-object-detection/run.py:115]: Where is this file?

Comment (freddyaboulton) [reply demo/rt-detr-object-detection/run.py:115]: My bad forgot to push.  Good call

File: guides/07_streaming/03_object-detection-from-video.md
[the remark this answers is not in the document]
Comment (freddyaboulton) [reply guides/07_streaming/03_object-detection-from-video.md:1]: Just added - good catch!

File: guides/07_streaming/02_object-detection-from-webcam-with-webrtc.md [hunk 2]
```diff
@@ -65,6 +65,8 @@ The Gradio demo is straightforward, but we'll implement a few specific features:
 We'll also apply custom CSS to center the webcam and slider on the page.
 
 ```python
+import gradio as gr
+
```
Comment (abidlabs) [inline guides/07_streaming/02_object-detection-from-webcam-with-webrtc.md:69]:
```suggestion
from gradio_webrtc import WebRTC

```

**Review by abidlabs — APPROVED** [review]
lgtm with the one addition above!

**Pull request merged**